src/main/java/com/stefancooper/SpigotUHC/events/BaseEvents.java

```diff
@@ -206,4 +226,196 @@ public void onChatMessage(AsyncPlayerChatEvent event) {
             event.setMessage(String.format("(Death Chat) %s", event.getMessage()));
         }
     }
-}
+
+    @EventHandler
+    public void onInventoryClick(InventoryClickEvent event) {

```

Comment:
can we move all of these new event handlers into the EnchantmentEvents?  All of these are specific for enchants